ngx_http_subs_filter_module.c

```diff
@@ -36,6 +36,8 @@ typedef struct {
     ngx_flag_t     has_captured;
```

Comment:
I like this feature. Thanks for your contribution. Could you add the  test script and docs?